lib/customs.js

```diff
@@ -22,7 +22,12 @@ module.exports = function (log, error) {
   function Customs(url) {
     if (url === 'none') {
       this.pool = {
-        post: function () { return P.resolve({ block: false })},
+        post: function (postUrl, postData) {
+          if (postUrl === '/check' && postData.action === 'accountLogin') {
```

Comment:
These threw me off some. Are these basically supposed to be mocks for running the test suite?